### PR TITLE
Improve "show" svg in case where there are many 0D Variables

### LINKED ISSUE #1310: "show()" visualisation is too cluttered with many scalar variables
Since Mantid workspace logs are now converted to separate scipp variables, we end up with many scalar variables in the visualisation when `show()` is called:
Screenshot at 2020-09-09 14-00-38

Suggested fix is to omit all but the first and last scalar variable block and separate them with an ellipsis.

### PR DESCRIPTION
**Pull request by matthew-d-jones**
Closes #1310 

Changes:
- If there are more than 5 0D Variables then only the first and last are drawn as "drawers" and an ellipsis shows that some were omitted.
- Titles longer than 13 characters are truncated to 13 characters and ".." appended.

Screenshot_20200914_160418

### CODE REVIEW

File: python/src/scipp/show.py [hunk 1]
```diff
@@ -405,25 +415,87 @@ def make_svg(self):
...
+                # Draw last variable
+                content, width, height, offset = _append_drawer(
+                    content, width, height, offset, layout_direction,
+                    last_item)
+            else:
+                if reverse:
+                    area = reversed(area)
+                for item in area:
+                    content, width, height, offset = _append_drawer(
+                        content, width, height, offset, layout_direction, item)
```
Comment (SimonHeybrock) [inline python/src/scipp/show.py:498]: Could this be simplified by conditionally replacing `aread[1:-1]` with the ellipses?

**Review by SimonHeybrock — COMMENTED** [review]
Builds are failing in the docs, but it looks like a broken Mantid package? @DanNixon any idea what this boost issue could be?

**Review by SimonHeybrock — APPROVED** [review]

**Review by SimonHeybrock — CHANGES_REQUESTED** [review]
Looking at the docs from the build artifacts: The rotation causes labels to exceed the top of the image (see "Data Structures" doc page). Maybe just set rotation to zero and think about a fix later, so we don't spend excessive amount of time on this, provided that most of the docs look ok?

**Comment by matthew-d-jones** [comment]
@SimonHeybrock Agreed. It's a shame; it looked good on the real files that I tried, but terrible on much of the docs.

**Comment by SimonHeybrock** [comment]
Something is not right, for variables it is truncating the dims/shape listed as "title"?

**Pull request merged**